lib/modules/zfs.go

```diff
@@ -0,0 +1,60 @@
+package modules
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/davidscholberg/go-i3barjson"
+	"os/exec"
+	"strings"
+)
+
+// Zfs represents the configuration data for the ZFS block
+type Zfs struct {
+	BlockConfigBase `yaml:",inline"`
+	PoolName        string `yaml:"zpool_name"`
+	ZpoolBin        string `yaml:"zpool_bin"`
+}
+
+// UpdateBlock updates the ZFS block
+func (c Zfs) UpdateBlock(b *i3barjson.Block) {
+	b.Color = c.Color
+	fullTextFmt := fmt.Sprintf("%s%s - %%s", c.Label, c.PoolName)
```

Comment:
I don't think that `c.PoolName` should be included in the output directly. It should be up to the user to determine if the full pool name or an abbreviated pool name should be in the output. Having only the `c.Label` field here allows the user to make such a choice.